docs/introduction/docker-compose.mdx

```diff
@@ -26,6 +26,6 @@ Run the following command to start the Pezzo stack:
 docker-compose up
 ```
 
-This command will spin up the all infrastructure services (PostgreSQL, Redis, etc.) and the Pezzo components (Server, Console). It will also automatically apply database migratiosn to the PostgreSQL database.
+This command will spin up the all infrastructure services (PostgreSQL, Redis, etc.) and the Pezzo components (Server, Console). It will also automatically apply database migration to the PostgreSQL database.
```

Comment:
```suggestion
This command will spin up all infrastructure services (PostgreSQL, Redis, etc.) and the Pezzo components (Server, Console). It will also automatically apply database migration to the PostgreSQL database.
```